### PR TITLE
Added translate via mt api and test case in MachineTranslationEnginesApis

### PR DESCRIPTION
**Pull request by s-aLman**

### CODE REVIEW

File: src/main/java/com/crowdin/client/machinetranslationengines/MachineTranslationEnginesApi.java [hunk 1]
```diff
@@ -77,4 +74,9 @@ public ResponseObject<MachineTranslation> editMt(Long mtId, List<PatchRequest> r
         MachineTranslationResponseObject machineTranslationResponseObject = this.httpClient.patch(this.url + "/mts/" + mtId, request, new HttpRequestConfig(), MachineTranslationResponseObject.class);
         return ResponseObject.of(machineTranslationResponseObject.getData());
     }
+
+    public ResponseObject<MtTranslateResponse> translateViaMt(Long mtId, MtTranslateRequest request) throws HttpException, HttpBadRequestException {
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/machinetranslationengines/MachineTranslationEnginesApi.java:78]: Please add an annotation here

Comment (s-aLman) [reply src/main/java/com/crowdin/client/machinetranslationengines/MachineTranslationEnginesApi.java:78]: sure.

Comment (s-aLman) [reply src/main/java/com/crowdin/client/machinetranslationengines/MachineTranslationEnginesApi.java:83]: done.

**Comment by andrii-bodnar** [comment]
@s-aLman thank you very much for your contribution! 🚀

**Review by andrii-bodnar — APPROVED** [review]

**Comment by s-aLman** [comment]
@andrii-bodnar Thank you for giving me the opportunity. I enjoyed working on all these enhancements. 😊 Looking forward to make more contributions.

**Comment by andrii-bodnar** [comment]
@s-aLman we are extremely happy with your contributions! ❤️ 

Just added one more enhancement: #107

Also, you can try the #94 if you wish 🙂

**Comment by andrii-bodnar** [comment]
Hi, @s-aLman, we appreciate your contributions to this project and would like to send you some gifts from Crowdin. Could you please reach me at `andrii.bodnar@crowdin.com` to discuss the delivery details?

**Comment by s-aLman** [comment]
@andrii-bodnar Sorry man, I think its very rude of me. I don't know why I said that 😅. I really want gifts from Crowdin. I hope this offer is still open 🙂. I will share my delivery details. Thank you.

**Comment by andrii-bodnar** [comment]
@s-aLman it's not a problem, anyway thank you 🙂

**Pull request merged**